web/src/lib/i18n/en.json

```diff
@@ -841,6 +842,7 @@
   "ok": "Ok",
   "oldest_first": "Oldest first",
   "onboarding": "Onboarding",
+  "onboarding_privacy_description": "To work properly, Immich needs to rely on external services such as a tile provider used for the map feature and Github to be aware of new releases",
```

Comment:
I think it is a bad idea to list specific features in this translation. That just means it will be wrong whenever we add another service. What about something like:

> The following features rely on external services to function properly:

If we add another service, we can just refer to new keys/translations in the card. We can also change/remove services as needed without requiring existing translations to be updated.